src/map/lapack2flash/FLASH_get_set_controls.c

```diff
@@ -0,0 +1,81 @@
+/*
+
+    Copyright (C) 2023, Advanced Micro Devices, Inc.
+
+    This file is part of libflame and is available under the 3-Clause
+    BSD license, which can be found in the LICENSE file at the top-level
+    directory, or at http://opensource.org/licenses/BSD-3-Clause
+
+*/
+
+#include "FLAME.h"
+
+#ifdef FLA_ENABLE_LAPACK2FLASH // Start lapack2flash
+
+static dim_t        flash_blocksize = 1024;
```

Comment:
given that this is going into FLAME.h and are static variables, maybe give more unique names like flash2lapack_$x ?

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/7/commits/cfec5dbecf82a0dbb46160d15d632f72c005f45e